DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Extensions/ElementExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using Autodesk.Revit.DB;
+
+namespace Speckle.Converters.RevitShared.Extensions;
+
+public static class ElementExtensions
+{
+  public static IList<ElementId> GetHostedElementIds(this Element host)
+  {
+    IList<ElementId> ids;
+    if (host is HostObject hostObject)
+    {
+      ids = hostObject.FindInserts(true, false, false, false);
+    }
+    else
+    {
+      var typeFilter = new ElementIsElementTypeFilter(true);
+      var categoryFilter = new ElementMulticategoryFilter(
+        new List<BuiltInCategory>()
+        {
+          BuiltInCategory.OST_CLines,
+          BuiltInCategory.OST_SketchLines,
+          BuiltInCategory.OST_WeakDims
```

Comment:
I'm assuming these have been ported from the original logic. I'd be curious to know if there is any others we should include here.